### PR TITLE
sys/shell: removed `route` to show the RPL internal routing table

### PR DESCRIPTION
**Pull request by BytesGalore**
With the introduction of the FIB, RPL will eventually (#2765) use it to store the discovered next-hops.
This PR removes the route command that accesses the RPL internal routing table.

### CODE REVIEW

File: sys/shell/commands/sc_fib.c [hunk 1]
```diff
@@ -71,7 +71,7 @@ int _fib_route_handler(int argc, char **argv)
     /* e.g. fibroute right now dont care about the adress/protocol family */
     if (argc == 1) {
         fib_print_routes();
-        return;
+        return 0;
```
Comment (BytesGalore) [inline sys/shell/commands/sc_fib.c]: The changes in this file are rather unrelated to the PR description, but they only fix a warning regarding return values. 
So if there are no objections I would leave them inside and not open a new PR.

**Comment by OlegHahm** [comment]
So, this should be merged after #2765, right? How about adding the corresponding dependency label?

**Comment by BytesGalore** [comment]
Basically its the other way around the other #2765 should be merged after this is merged since I want to remove [this](https://github.com/BytesGalore/RIOT/commit/b555f6057f5548691e043afb4f5e6e3050b9fd3b) commit :) I added a label there

**Comment by OlegHahm** [comment]
Nevertheless they should be merged more or less at the same time to ensure that we don't have a state without a route lookup command for a long time.

**Comment by BytesGalore** [comment]
@OlegHahm definitely, a short time distance between the both would be very beneficial :)

**Comment by cgundogan** [comment]
ACK as soon as #2765 has been reviewed and ACked, too

File: sys/shell/commands/sc_fib.c (hunk 1 above)
Comment (BytesGalore) [inline sys/shell/commands/sc_fib.c]: moved it to a separate commit (#2812)

**Comment by cgundogan** [comment]
this PR is obsolete now, because it references the former nework stack
